src/receipt/_style.scss

```diff
@@ -0,0 +1,67 @@
+#receipt-container {
+    width: 1170px;
+    padding-top: 2rem;
+}
+
+.page__receipt {
+    h2 {
+        margin-bottom: 10px;
+        font-weight: 400;
+    }
+
+    .thank-you {
+        font-size: 38px;
```

Comment:
[curious] Why 38px here? It's not a font size in our design system's type ramp. See https://paragon-openedx.netlify.app/foundations/typography.

If we do need a custom font size here, we should use convert this to `rem`.

Reply:
The aforementioned px specific size 😅 - will update to Paragon friendly size! Thanks for asking!

Reply:
And preferably using a SCSS variable or CSS utility class, too (not hardcoded) 😉